swifty/swifty_manage.py

```diff
@@ -324,12 +353,14 @@ def main():
     if args.publish is not None:
         return publish(args.publish)
 
+    move_optionals()
```

Comment:
I would prefer if this is set as a flag (enables the user to only copy the optionals), and make it a default operation when "repo" argument is provided. Additionally, I would like to see a flag --no-optionals, which on the contrary would NOT copy optionals when building a repository (just in case this behaviour is needed at any time).

Reply:
If said ideas are implemented, this behavior has to be noted on the argparse help output.

Reply:
> I would prefer if this is set as a flag (enables the user to only copy the optionals), and make it a default operation when "repo" argument is provided.

That's inverted behaviour dependant on another argument then.

> Additionally, I would like to see a flag --no-optionals, which on the contrary would NOT copy optionals when building a repository (just in case this behaviour is needed at any time).

Nothing will work in that case though, because Swifty will not be able to build. The proper way to avoid optionals is to delete them from the `@mod/` folder to avoid copying/moving.

---

What I would agree with, is to move optionals only when a repo is provided. That way just running `python swifty_manage.py` would do nothing, while providing a repo would do move (required anyways) and build. In that case `python swifty_manage.py --only-optional` would make sense.